core/java/iesi-core/src/main/resources/log4j2.xml

```diff
@@ -139,7 +139,7 @@
             <AppenderRef ref="queries" level="all" />
         </Logger>
         <Logger name="io.metadew.iesi.script" additivity="false">
-            <AppenderRef ref="STDOUT" level="info" />
+            <AppenderRef ref="STDOUT" level="debug" />
```

Comment:
Why is this changed to debug?

Reply:
correct, temporarily set for debugging issues, will be set correctly in next commit